llvm/docs/ReleaseNotes.md

```diff
@@ -116,6 +116,8 @@ Changes to the LLVM IR
 Changes to LLVM infrastructure
 ------------------------------
 
+ * Several methods that use Instruction pointers as insertion positions (moveBefore, getFirstNonPHI) have been deprecated in favour of overloads and variants that use `BasicBlock::iterator`s instead. The instruction-flavoured methods will be removed in a future release. This work is part of the [RemoveDIs](https://llvm.org/docs/RemoveDIsDebugInfo.html) project, the documentation for which contains instructions for updating call-sites using the deprecated methods.
```

Comment:
> Couple of nits from me (sorry).

... that github ate.

I think there's some unnecessary ambiguity around whether `moveBefore` + `getFirstNonPHI` is the complete list of affected functions or not (I think both the word "several" and then having those two in parens is causing it). Can we be more precise?

Secondly, IMO I think changing "instruction-flavoured" to "pointer-flavoured" slightly improves clarity. YMMV though.

Reply:
Added some rewording to "Two methods", "pointer-flavoured", and stating that the methods use pointers as "code positions" because technically `getFirstNonPHI` isn't about insertion.

(It's a bit tricky to correctly abstract the wording over "this is about insertions but not all the functions are inserters".)